package.json

```diff
@@ -11,18 +11,25 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@angular/common": "=2.0.0-rc.3",
-    "@angular/compiler": "=2.0.0-rc.3",
-    "@angular/core": "=2.0.0-rc.3",
-    "@angular/http": "=2.0.0-rc.3",
-    "@angular/platform-browser": "=2.0.0-rc.3",
-    "@angular/platform-browser-dynamic": "=2.0.0-rc.3",
     "es6-shim": "^0.35.0",
-    "reflect-metadata": "0.1.2",
-    "rxjs": "=5.0.0-beta.6",
-    "systemjs": "0.19.25",
     "ui-router-ng2": ">=1.0.0-beta.1",
-    "zone.js": "^0.6.12"
+    "@angular/common": "2.0.0-rc.5",
+    "@angular/compiler": "2.0.0-rc.5",
+    "@angular/core": "2.0.0-rc.5",
+    "@angular/forms": "0.3.0",
+    "@angular/http": "2.0.0-rc.5",
+    "@angular/platform-browser": "2.0.0-rc.5",
+    "@angular/platform-browser-dynamic": "2.0.0-rc.5",
+    "@angular/router": "3.0.0-rc.1",
```

Comment:
These are part of the angular2 hero tutorial which was adding these, you will find the in the polyfill.js. you will also find that without zone, angular2 rc5 throws error.